app/src/main/kotlin/org/kotlinlsp/index/ScanFilesThread.kt

```diff
@@ -1,38 +1,46 @@
 package org.kotlinlsp.index
 
-import com.intellij.openapi.vfs.VirtualFile
 import org.kotlinlsp.analysis.modules.Module
 import org.kotlinlsp.analysis.modules.getModuleList
-import org.kotlinlsp.common.info
 import org.kotlinlsp.index.worker.WorkerThread
 import java.util.concurrent.atomic.AtomicBoolean
 
 class ScanFilesThread(
     private val worker: WorkerThread,
     private val rootModule: Module
-): Runnable {
+) : Runnable {
     private val shouldStop = AtomicBoolean(false)
 
     override fun run() {
         var gotFirstLibraryFile = false
 
         rootModule.getModuleList()
-            .sortedByDescending { it.isSourceModule }   // Index source files first so initial analysis takes less time
+            .sortedByDescending { it.isSourceModule }   // Scan source files first so initial analysis takes less time
```

Comment:
For the "scan phase" we only need to scan source files, so we can filter here by `it.isSourceModule`